docs/source/autograd.rst

```diff
@@ -50,6 +50,10 @@ you can use it as ``functional.jacobian(lambda x: f(x, constant, flag=flag), inp
 Locally disabling gradient computation
```

Comment:
Maybe this is also a good place to place such a note

Reply:
tbh I don't know if many people end up reading this section.

Reply:
true